src/Services/Account/Command/Application/UseCases/Events/RebuildProjectionRequestedInteractor.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Application.Abstractions;
+using Application.Services;
+using Contracts.DataTransferObjects;
+using Contracts.Services.Account;
+using Domain.Aggregates;
+using MassTransit;
+
+namespace Application.UseCases.Events;
+
+public class RebuildProjectionRequestedInteractor : IInteractor<NotificationEvent.RebuildProjectionRequested>
+{
+    private readonly IApplicationService _applicationService;
+
+    public RebuildProjectionRequestedInteractor(IApplicationService applicationService)
+    {
+        _applicationService = applicationService;
+    }
+
+    public async Task InteractAsync(NotificationEvent.RebuildProjectionRequested @event, CancellationToken cancellationToken)
+    {
+        var account = (Account)await _applicationService.LoadAggregateAsync<Account>(@event.AccountId, cancellationToken);
```

Comment:
The CAST is not necessary because the generics already define the Aggregate type